client/utils/guidance.py

```diff
@@ -218,10 +233,11 @@ def set_tg(self,access_token,guidance_flag):
                 return EMPTY_MISUCOUNT_TGID_PRNTEDMSG
 
             try:
-                data = json.loads(urlopen(TGSERVER + cur_email + TG_SERVER_ENDING,timeout =1).read().decode("utf-8"))
+                data = json.loads(urlopen(TGSERVER + cur_email +"/" + self.assignment + TG_SERVER_ENDING,timeout =1).read().decode("utf-8"))
             except IOError as e:
                 data = {"tg":-1}
```

Comment:
Seems worthwhile to log that there was an IOError communicating to the server (again `.warning`)